docs/docs.go

```diff
@@ -0,0 +1,268 @@
+                    "type": "string"
+                },
+                "cmd_status": {
+                    "type": "string"
+                },
+                "cmd_stop": {
+                    "type": "string"
+                },
+                "config_args": {
+                    "$ref": "#/definitions/models.L3afDNFArgs"
+                },
+                "config_file_path": {
+                    "type": "string"
+                },
+                "cpu": {
+                    "type": "integer"
+                },
+                "id": {
+                    "type": "integer"
+                },
+                "is_plugin": {
+                    "type": "boolean"
+                },
+                "map_args": {
+                    "$ref": "#/definitions/models.L3afDNFArgs"
+                },
+                "map_name": {
+                    "type": "string"
+                },
+                "memory": {
+                    "type": "integer"
+                },
+                "monitor_maps": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/models.L3afDNFMetricsMap"
+                    }
+                },
+                "name": {
+                    "type": "string"
+                },
+                "prog_type": {
+                    "type": "string"
+                },
+                "rules": {
+                    "type": "string"
+                },
+                "rules_file": {
+                    "type": "string"
+                },
+                "seq_id": {
+                    "type": "integer"
+                },
+                "start_args": {
+                    "$ref": "#/definitions/models.L3afDNFArgs"
+                },
+                "status_args": {
+                    "$ref": "#/definitions/models.L3afDNFArgs"
+                },
+                "stop_args": {
+                    "$ref": "#/definitions/models.L3afDNFArgs"
+                },
+                "user_program_daemon": {
+                    "type": "boolean"
+                },
+                "version": {
+                    "type": "string"
+                }
+            }
+        },
+        "models.BPFPrograms": {
+            "type": "object",
+            "properties": {
+                "tc_egress": {
+                    "description": "list of tc egress bpf programs",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/models.BPFProgram"
+                    }
+                },
+                "tc_ingress": {
+                    "description": "list of tc ingress bpf programs",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/models.BPFProgram"
+                    }
+                },
+                "xdp_ingress": {
+                    "description": "list of xdp bpf programs",
```

Comment:
Ok, but maybe better to be consistent between 207 and 208... either "xdp" or "xdp ingress" instead of one of each